docs/rules/jsx-no-undef.md

```diff
@@ -18,6 +18,52 @@ var Hello = require('./Hello');
 <Hello name="John" />;
 ```
 
+## Rule Options
+
+```js
+...
+"jsx-no-undef": [<enabled>, { "importedOnly": <boolean> }]
+...
+```
+
+### `importedOnly`
```

Comment:
That seems like it'd be a breaking change - an option now, flipping the default later, seems safer